Base/QTGUI/qSlicerScriptedLoadableModuleFactory.cxx

```diff
@@ -77,6 +61,22 @@ qSlicerAbstractCoreModule* ctkFactoryScriptedItem::instanciator()
     }
 #endif
 
+  return true;
+}
+
+//----------------------------------------------------------------------------
+qSlicerAbstractCoreModule* ctkFactoryScriptedItem::instanciator()
```

Comment:
Can you please rename this function? I think there it is misspelled (I know it was the same before, but now that you are here...). Thanks!
https://en.wiktionary.org/wiki/instantiator